website/blog/2024-02-11-FSM-GroupChat/index.mdx

```diff
@@ -0,0 +1,288 @@
+    Planner (to chat_manager):
+
+    2
+
+    --------------------------------------------------------------------------------
+    Engineer (to chat_manager):
+
+    3
+
+    --------------------------------------------------------------------------------
+    Executor (to chat_manager):
+
+    4
+
+    --------------------------------------------------------------------------------
+    Engineer (to chat_manager):
+
+    5
+
+    --------------------------------------------------------------------------------
+    Critic (to chat_manager):
+
+    6
+
+    --------------------------------------------------------------------------------
+    Engineer (to chat_manager):
+
+    7
+
+    --------------------------------------------------------------------------------
+    Critic (to chat_manager):
+
+    8
+
+    --------------------------------------------------------------------------------
+    Engineer (to chat_manager):
+
+    9
+
+    --------------------------------------------------------------------------------
+    Executor (to chat_manager):
+
+    10
+
+    --------------------------------------------------------------------------------
+    Engineer (to chat_manager):
+
+    11
+
+    --------------------------------------------------------------------------------
+    Critic (to chat_manager):
+
+    12
+
+    --------------------------------------------------------------------------------
+    Engineer (to chat_manager):
+
+    13
+
+    --------------------------------------------------------------------------------
+    Critic (to chat_manager):
+
+    14
+
+    --------------------------------------------------------------------------------
+    Engineer (to chat_manager):
+
+    15
+
+    --------------------------------------------------------------------------------
+    Executor (to chat_manager):
+
+    16
+
+    --------------------------------------------------------------------------------
+    Engineer (to chat_manager):
+
+    17
+
+    --------------------------------------------------------------------------------
+    Critic (to chat_manager):
+
+    18
+
+    --------------------------------------------------------------------------------
+    Engineer (to chat_manager):
+
+    19
+
+    --------------------------------------------------------------------------------
+    Critic (to chat_manager):
+
+    20
+
+    --------------------------------------------------------------------------------
+    Planner (to chat_manager):
+
+    TERMINATE
+    ```
+
+## Notebook examples
+More examples can be found in the [notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_graph_modelling_language_using_select_speaker.ipynb). The notebook includes more examples of possible transition paths such as (1) hub and spoke, (2) sequential team operations, and (3) think aloud and debate. It also uses the function `visualize_speaker_transitions_dict` from `autogen.graph_utils` to visualize the various graphs.

```

Comment:
Thank you for the suggestion. I have cloned the SoM notebook and copied the contents over, so that the web-rendered version becomes possible. 
I have also renamed the notebook name/title/description to include FSM.

Reply:
How about using the new link here?

Reply:
Yes ok. Thank you.